### ISSUE TITLE
UWP PeerCC to PeerCC-browser streaming issue

### ISSUE THREAD

**Issue by Muhayyuddin** [webrtc-uwp/webrtc-uwp-sdk#197]
Hi, I would like to do a video call from UWP PeerCC app to chrome browser or from HL peerCC (unity) to browser.
with PeerCC-browser [example](https://github.com/OpticalTone/PeerCC-Browser), I am able to call successfully from web-browser to web-browser. but when I try to call from UWP peerCC app, I am not able to see the remote stream at browser.
browser --> browser : works fine
browser --> uwp peerCC : at peercc i can see local and remote video but on browser side it just show local video.
HL --> browser same result.
it also did not show any error.

With some further investigation I found that
the function ontrack (on browser side) called but event.stream is null. it is not receiving the remote stream.
```javascript
    //display remote track in the video element
    pc.ontrack = function(event) {
      videoRenderer = document.getElementById("rtcRenderer");
        videoRenderer.srcObject = event.streams[0];
      console.log("Remote track resceved");
      document.body.style.cursor = "default";
    };
    updateServerStatus();
  }
```
I have tried with **webRTC native peerconnection client example (m70)** it works fine I can call peerconnection_client to browser. 

Any suggestions or potential solution, what should be the potential issue what should I modify in PeerCC app example.

any potential solution or suggestion to figure it out (why remote video form peercc is not appear at browser) will be really appreciated.

I am using M71-unity-fix branch.

thank you.

**Comment by Muhayyuddin** [comment]
@jamescadd  Thank you for your reply, 
I am using  **Chrome Version 77.0.3865.120 (Official Build) (64-bit)** I have also tried with MS Edge  the same result.  Do you think it could be because of codec?

**Comment by jamescadd** [comment]
You could try a few codecs and see, but I think there is a more basic issue in the UWP stack there.  Apologies for not having more for you to go on, the WebRTC traces in Chrome may be a place to start.

**Comment by Muhayyuddin** [comment]
Thank you @jamescadd i will try to figure it out and will share if I found something interesting towards the solution of this issue.

**Comment by Muhayyuddin** [comment]
@jamescadd  I have tried with the older version of chrome (Version 71.0.3578.80 (Official Build) (64-bit)) it works perfectly fine. I am able to call from peerCC UWP app to browser also from HoloLens to browser.  
I think there is a compatibility issue between webRTC M71 and the webRTC branch in the current version of chrome (that is I think M77 or M75).  
If I found some free time I will try with UWP PeerCC M75 to browser (latest version) in next few days. The only issue (that i experienced in the past) was peerCC unity for HoloLense do not work properly with M75 (I was not able to initiate the call from HoloLens to peerCC UWP).  I dont know if this issue is fixed or still there?

**Comment by LoadLibrary** [comment]
WebRTC-UWP is based upon the WebRTC.org code base shipped within Chrome. AFAIK, the WebRTC.org code base changed significantly between m71 and m75. Please take a look at the deprecation section of the WebRTC.org release notes (https://webrtc.org/release-notes/). Have you tried to run the PeerCC-browser using two different versions of Chrome matching the ones you're using for WebRTC-UWP?

**Comment by Muhayyuddin** [comment]
@LoadLibrary Yes I have tried calling from Chrome 71 to Chrome 78 using PeerCC-Browser it works fine. but when I try to call form peerCCUWP to PeerCC-browser, it works PeerCCUWP to chrome 71 and does not work with chrome 78 (chrome send stream successfully but receive empty remote stream).
